src/relay.cpp

```diff
@@ -401,12 +406,7 @@ const struct dhcpv6_option *parse_dhcpv6_opt(const uint8_t *buffer, const uint8_
 }
 
 void sent_msg_increase_counter(relay_config *config, uint8_t msg_type) {
```

Comment:
Why do we need this method? i think `increase_counter `can replace `sent_msg_increase_counter` method now.